### DISCUSSION TITLE
Add "MA" (or something) prefix to Music Assistant media players

### DISCUSSION THREAD

**Discussion by chrkov** [music-assistant/support#1368] [Feature requests and ideas]
For sorting and filtering purposes, add a prefix to the media players that MA creates.   Right now, MA just creates a 'media_player.something_2'  for 'media_player.something'    It would be nice to see it as 'media_player.MA_something' instead.   This way you can filter on 'media_player.MA_*'.

My emby integration does exactly this.   'media_player.basement_tv' emby makes its player 'media_player.emby_basement_tv' which works great for filtering and grouping.

**Comment by Mr-HaleYa** [comment]
This would actually be really nice. 
Currently, my Tv has created media_player.tv, media_player.tv_1, media_player.tv_2 due to having multiple casting technologies in it so now Music Assistant has made media_player.tv_3, and its all rather confusing/messy.

**Reply by OzGav** [reply]
You can find all the created players by going to settings>>integrations>>music assistant. If there is a compelling reason why this is not adequate then please provide a detailed use case and we can have a look at it.

**Reply by Mr-HaleYa** [reply]
I agree, there's nothing wrong with the way it works now, it just looks less than it could. I've gone through and renamed all my entities MA created with a `ma_` prefix eg `media_player.ma_kitchen_display`. Look better/cleaner and at least 3 others agree 😅
When creating automation or editing a card I can just type `media_player.ma_` and they all pop up, feels convenient, more refined, less slapped on +1 suffixy.

**Reply by Sireone** [reply]
This really should be a native feature of Music Assistant.  I just added the Mediocre Media Player card and trying to select my MA entities is a roll of the dice as they are duplicated.

**Comment by marcelveldt** [comment]
We can't predefine entity id's without also prefixing the entire name of the player.
Also I do not really see the point of this as MA now works like ANY other HA integration and you can easily change the entity id's yourself.
Note that the entire foundation of MA changed. Where in v1 we based our players on the HA players it is now the other way around. You will have plenty of users that will only have a single entity for a player because the player was originated from MA.

Mass players are easily identified because they have a 'mass_player_id' attribute.

**Reply by Sireone** [reply]
Bringing this topic back to life.  I really love Music Assistant, really do, but this is still coming back to bite me, specifically when I'm now leveraging AI via OpenCode.  Is that prefix really not an option in the future (willing to beta test, LOL)?  Could even make it an 'opt-in' feature where users can either keep or rename their entities.  It seems that as the OP mentioned, this is currenly being done in Emby (PR below), which only gets applied to new entiites.  Thoughts?

https://github.com/home-assistant/core/pull/32183

**Reply by OzGav** [reply]
Nothing has changed in the last three years. You can change the entity id yourself.
